tests/bug-979.sil

```diff
@@ -0,0 +1,8 @@
+\begin{document}
```

Comment:
```suggestion
\begin[papersize=a6]{document}
\nofolios
\noindent
```

Killing off the folios when not specifically testing frames saves quite a few false positives. Using a small paper size by default makes visually reviewing them easier when mass updates change something and they have to be inspected. A6 is what I've standardized on when papersize isn't important for any other reason. Killing off indentation just make inspection a tad easier.